app/src/main/java/io/github/sds100/keymapper/mappings/keymaps/KeyMapField.kt

```diff
@@ -0,0 +1,19 @@
+package io.github.sds100.keymapper.mappings.keymaps
+
+import io.github.sds100.keymapper.constraints.KeyMapConstraintsComparator
+import io.github.sds100.keymapper.mappings.keymaps.trigger.KeyMapTriggerComparator
+
+enum class KeyMapField {
```

Comment:
I would move this to the `sorting` package and rename it SortField? I could see one adding more ways to sort that aren't strictly key map fields?